CHANGES.rst

```diff
@@ -1,3 +1,15 @@
+2.3.0 (March 01, 2023)
+======================
+New feature release in the 2.3.x series.
+
+This release supports fMRIPrep 23.0.x and Nibabies 23.0.x.
```

Comment:
@mgxd Is this a correct line from the nibabies perspective?

Reply:
We've already released 23.0.0, but whether next release is 23.0.1 or 23.1.0, and because we talked about lessening version restrictions, this release can be incorporated

Reply:
K.